Dockerfile

```diff
@@ -1,17 +1,21 @@
-FROM golang:1.11 AS build
+FROM golang:alpine AS build
 
-COPY ./ /goproxy
+RUN apk add --no-cache -U git make build-base
```

Comment:
set `CGO_ENABLED=0` will solve 
```
# net
exec: "gcc": executable file not found in $PATH
```
then no need to install `build-base`

Reply:
Okay one moment...

Reply:
Done and verified:

```#!bash
$ dki -t --rm --entrypoint /bin/sh goproxyio/goproxy
/ # ldd /goproxy
ldd: /goproxy: Not a valid dynamic program
```

Reply:
emmm, set it just in dockerfile instead makefile maybe better. :smile:

Reply:
Okay :)

Reply:
And done :)

```#!bash
prologic@Jamess-MacBook
Thu Jan 10 17:52:36
~/Contributions/goproxy
 (master) 2
$ make image && dki -t --rm --entrypoint /bin/sh goproxy/goproxy -c 'ldd /goproxy'
Sending build context to Docker daemon   15.7MB
Step 1/13 : FROM golang:alpine AS build
 ---> f56365ec0638
Step 2/13 : RUN apk add --no-cache -U make
 ---> Using cache
 ---> cd522af6f9c7
Step 3/13 : COPY . /src/goproxy
 ---> Using cache
 ---> 34712be82656
Step 4/13 : WORKDIR /src/goproxy
 ---> Using cache
 ---> ca221399ac7b
Step 5/13 : ENV CGO_ENABLED=0
 ---> Using cache
 ---> 4d09db4abb19
Step 6/13 : RUN make
 ---> Using cache
 ---> 602a911a9722
Step 7/13 : FROM alpine:latest
 ---> 196d12cf6ab1
Step 8/13 : RUN apk add --no-cache -U git mercurial subversion bzr fossil
 ---> Using cache
 ---> da41a94e86f0
Step 9/13 : COPY --from=build /src/goproxy/goproxy /goproxy
 ---> Using cache
 ---> f95a474203a9
Step 10/13 : VOLUME /go
 ---> Using cache
 ---> 6e61f934e40b
Step 11/13 : EXPOSE 8081
 ---> Using cache
 ---> 62d2c56e6271
Step 12/13 : ENTRYPOINT ["/goproxy"]
 ---> Using cache
 ---> 66904c6a8f96
Step 13/13 : CMD []
 ---> Using cache
 ---> 0254ba7b945b
Successfully built 0254ba7b945b
Successfully tagged goproxy/goproxy:latest
ldd: /goproxy: Not a valid dynamic program
```